pillarbox-player/src/main/java/ch/srgssr/pillarbox/player/PillarboxPlayer.kt

```diff
@@ -181,6 +181,15 @@ fun Player.getPlaybackSpeed(): Float {
     return playbackParameters.speed
 }
 
+/**
+ * Current position percent
+ *
+ * @return the current position in percent [0,1].
+ */
+fun Player.currentPositionPercent(): Float {
```

Comment:
Percentage? Or are they other examples in official APIs where such a method would be called Percent?

Reply:
Yes it is`Player.bufferedPercentage` to get the buffering percentage. It is a int between [0,100|. So I guess we can rename it to `currentPositionPercentage`.

Reply:
You should likely rename other occurrences of `percent` as `percentage` then 😇

Have fun! 👺